lib/features/jobs_list/domain/usecases/get_jobs_list.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:dartz/dartz.dart';
+import 'package:jobs_tdd/cores/errors/failure.dart';
+import 'package:jobs_tdd/cores/usecases/usecase.dart';
+import 'package:jobs_tdd/features/jobs_list/data/models/data_model.dart';
+import 'package:jobs_tdd/features/jobs_list/domain/entities/data.dart';
+import 'package:jobs_tdd/features/jobs_list/domain/entities/job.dart';
+import 'package:jobs_tdd/features/jobs_list/domain/repositories/jobs_repository.dart';
+
+class GetJobList  {
```

Comment:
Use the naming convention for use case classes and file names:
GetJobsListUseCase
get_jobs_list_use_case.dart